multiscene/Readme.md

```diff
@@ -0,0 +1,38 @@
+# Multiscene Example
+
+This example demonstrates the usage of multiscene feature in Wonderland Engine version 1.2.0.
+
+## Overview
+
+Wonderland Engine previously supported managing only a single scene. With the new multiscene feature, you can now easily manage multiple scenes and share resources between them.
+
+## Usage
+
+(in the index.js)
+
+```javascript
+engine.loadSceneGroup('myNewScene.bin');
+```
+Scene Activation
+
+```js
+const myNewScene = this.engine.getSceneGroup(1).getScene(0);
```

Comment:
Maybe do not show `getSceneGroup` here, it's useful only if you didn't keep the group reference:

```js
const myNewScene = sceneGroup.getScene(0);
```

Reply:
but w.r.t the previous line , if we did 
```javascript
const sceneGroup = engine.loadSceneGroup('myNewScene.bin');
```
on index.js it wont be accessible over component right ?

Reply:
It's accessible, every scene root loaded is stored in the engine in case you don't want to store it yourself. In this example here, not need to read it from the engine

Reply:
great , i think it would we worth adding in the readme as well